prisma-fmt/src/get_dmmf.rs

```diff
@@ -0,0 +1,36 @@
+use serde::Deserialize;
+
+/// The implementation of the CLI getDmmf() utility and its JSON format.
+pub mod internal {
+    pub use dmmf::{dmmf_json_from_schema as get_dmmf, *};
+}
```

Comment:
Could this be replaced with just `use dmmf::*`? This is not a pattern we use anywhere else.

Reply:
Ok